src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -785,8 +785,8 @@ class Definitions {
     EmptyPackageVal,
     OpsPackageClass)
 
-    /** Lists core methods that don't have underlying bytecode, but are synthesized on-the-fly in every reflection universe */
-    lazy val syntheticCoreMethods = AnyMethods ++ ObjectMethods ++ List(String_+)
+  /** Lists core methods that don't have underlying bytecode, but are synthesized on-the-fly in every reflection universe */
+  lazy val syntheticCoreMethods = AnyMethods ++ ObjectMethods ++ List(String_+, throwMethod)
```

Comment:
indentation here was intentional: member definitions are indented with 2 more spaces compared to class definitions.